modules/docs/arrow-docs/docs/_includes/_sidebar-doc-versions.html

```diff
@@ -32,10 +32,9 @@
       </li>
 
       <li>
-        <a href="{{ previous_version.url | append: 'docs/' }}">
+        <a href="{{ previous_version.url | append: doc-link }}">
```

Comment:
We should remove the append here since the url that is going to build would be `https://arrow-kt.io/docs/0.9/core/` in the case of core, and that url doesn´t exist in that version.